### DISCUSSION TITLE
True 3D Chars

### DISCUSSION THREAD

**Discussion by JPC16** [ikemen-engine/Ikemen-GO#2225] [Feature Request]
Hi, Is it possible to add support for 3D models for the chars. Because the 3D chars for MUGEN/IKEMEN are just cropped images frame by frame and therefore have a poorer quality and are very pixelated. Compare to how are the chars based on Dragon Ball Fighterz and compared with the 3D models so you know what I'm talking about.

Adding 3D model support for the chars could save that tedious need to be cropping images frame by frame. Well I'm waiting for someone to make in IKEMEN GO a kind of Tekken or Virtua Fighter, although I don't know if it will be possible or not to implement a full 3D gameplay in IKEMEN GO.

**Comment by SuperFromND** [comment]
As far as I know nobody's planning to add support for this right now, sorry!

**Reply by JPC16** [reply]
What a shame. I would like to have a dream 3D MUGEN with characters from Virtua Fighter, Tekken, Soul Calibur, Dead or Alive, Bloody Roar, the PS2 Mortal Kombat games and the Dragon Ball Z Budokai Tenkaichi games. Considering that currently there are almost no 3D fighting games coming out, Tekken 8 and Dragon Ball Sparking Zero being the only recent 3D fighting games so far.

**Comment by JPC16** [comment]
While researching online, I discovered that there was a MUGEN clone that attempted to implement chars with real 3D models called MMF. Perhaps you could use it as a reference to implement real 3D chars in IKEMEN GO. I will also include a download link on Mediafire for all the builds I have collected so that they can be preserved.
https://mmfdevblog.wordpress.com/
https://www.mediafire.com/folder/fmpbc160knobz/MMF+(3D+MUGEN)#fmpbc160knobz

**Comment by K4thos** [comment]
@JPC16 where exactly did you find this blog and download the releases from? Google isn’t returning anything about it. Was the source code ever released?

**Reply by JPC16** [reply]
I found out about this Mugen clone because someone on the Ikemen Discord server posted a link to the blog. Just search for “MMF” on that server and you'll find the link.

**Reply by LeoKids** [reply]
By default MMF will return Multimedia Fusion,not the Mugen clone also called MMF...

**Reply by JPC16** [reply]
> By default MMF will return Multimedia Fusion,not the Mugen clone also called MMF...

I said you should search on Ikemen's Discord server. I didn't say you could search for it on Google, since it's very hard to find there.

**Comment by JPC16** [comment]
Hello again. It turns out that a user named homelab90 has managed to implement true 3D chars in Ikemen GO. I’ll post the video below. 
https://www.youtube.com/watch?v=YYtgRvyGqiY

You should contact this person to invite them to join the Ikemen GO development team and make this an official feature.
